py/tools/pex/main.py

```diff
@@ -159,7 +159,7 @@ def __call__(self, parser, namespace, value, option_str=None):
 ]
 
 for dep in options.dependencies:
-    dist = Distribution.load(dep + "/../")
+    dist = Distribution.load(dep)
```

Comment:
As i remember, Distribution.load has always wanted to read a dist-info, how does this work now?

Reply:
It should be functionally equivalent to the previous version I believe, but I might be missing something.  `_map_srcs` now emits the path up to  (and including) 'site-packages', instead of 'site-packages/some-package', so `+/../` isn't needed anymore

Reply:
right, `options.dependencies` in fact  just site-packages due to `/../`, so this change ought to be functionally equivalent. I am just curious if pex is able to package bare deps like this one where there is no directory but a single file.